repos/system_upgrade/el7toel8/actors/restrictedpcisscanner/libraries/restrictedpcisscanner.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+
+import urllib3
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.config import get_env
+from leapp.libraries.stdlib import api
+from leapp.models import RestrictedPCIDevices
+
+try:
+    # python3
+    from json import JSONDecodeError  # pylint: disable=ungrouped-imports
+except ImportError:
+    # python2
+    JSONDecodeError = ValueError
+    FileNotFoundError = IOError  # pylint: disable=redefined-builtin
+
+REQUEST_TIMEOUT = 0.5  # TODO Consider moving it to the leapp env var
+UNSUPPORTED_DRIVER_NAMES_GLOBAL_FILE = os.path.join(
+    "/etc/leapp/files", "unsupported_driver_names.json"
+)
+UNSUPPORTED_PCI_IDS_GLOBAL_FILE = os.path.join(
+    "/etc/leapp/files", "unsupported_pci_ids.json"
+)
+
+# TODO Once the microservice for providing the data about removed hardware
+#   will be provisioned to the Red Hat costumers, then this variable default
+#   value should be overwritten
+PCIS_HOST_DEFAULT = ""
+API_VERSION = "v1"
+
+
+def get_restricted_pcis(http, update_fallback_data=False):
```

Comment:
no more fallback, the flag doesn't make sense anymore

Reply:
resolved in https://github.com/oamg/leapp-repository/pull/602/commits/06e630c346b953bbe1dc9d26628be33ae194abfa